solidity/test-utils/deployERC721.ts

```diff
@@ -0,0 +1,48 @@
+import { Gravity } from "../typechain/Gravity";
+import { TestERC20A } from "../typechain/TestERC20A";
+import { GravityERC721} from "../typechain/GravityERC721";
+import { TestERC721A } from "../typechain/TestERC721A";
+import { ethers } from "hardhat";
+import { makeCheckpoint, getSignerAddresses, ZeroAddress } from "./pure";
+import { Signer } from "ethers";
+
+type DeployContractsOptions = {
+  corruptSig?: boolean;
```

Comment:
we're not handling signatures so we don't need corruptSig

Reply:
In fact I don't think we need this file at all, why not deploy the GravityERC721 in the existing deployContracts function and then just call that in `solidity/test/sendERC721ToCosmos.ts`

Reply:
Would prefer to use interface segregation here instead of overstuffing the index.ts
I extended index.ts in deployERC721.ts and then added the additional ERC721 deploys there